types/go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/aquasecurity/tracee/types
 
-go 1.21
+go 1.22.0
```

Comment:
Shouldn't we upgrade to go1.22.3 considering the toolchain directive below?

Reply:
Good question, it was my first change - if go and toolchain are the same, the toolchain will be omitted.

But I'm trying to establish a pattern of updates by saying that we are migrating to 1.22.0 (a newer version of go), while at the same time enforcing the use of a new version in the development environment and CI/CD pipelines without requiring immediately all users or environments upgrade - toolchain configuration will automatically download and use it for building without changing the user's environment.

This allows us to bring in a new toolchain with each patch release and ensures that the codebase remains compatible with older versions during the migration process.

Reply:
Ok, sounds good to me.